### PR TITLE
Initial content for the Great Sand Dunes

### PR DESCRIPTION
**Pull request by retlehs**
[saw your tweet](https://twitter.com/contrahacks/status/721181241983246336) about this project so figured i'd push some content for a place i've been to. 

since there's no content in this repo right now/no guidelines, i'm not exactly sure what you're looking for. i can update this PR with any feedback you have 🌄

### CODE REVIEW

File: north_america/united_states/colorado/parks/great_sand_dunes.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Great Sand Dunes National Park and Preserve
+
+[Great Sand Dunes National Park and Preserve](https://www.nps.gov/grsa/index.htm) is a United States National Park located in South-Central Colorado.
+
+The park contains the tallest dunes in North America, with Star Dune rising over 750 feet from top to bottom.
+
+## Scenic backcountry route
+
+Have a high-clearance 4WD vehicle? Take Medano Pass (9,931ft elevation), a scenic off-road trail that passes through the dunes. The 22 mile road includes soft sand for about 4 miles, 9 creek crossings, and a rocky roadbed near the summit of the pass.
```
Comment (yocontra) [inline north_america/united_states/colorado/parks/great_sand_dunes.md]: got any info on where to rent a high-clearance 4wd nearby or at the closest intl airport? i know hertz usually has some jeeps

**Comment by yocontra** [comment]
Just off the top of my head, I think for parks the format should be:

``` md
# Name of Park

![super cool park image to sell them](super-cool-park.jpeg)

## Why?

Because this park is sick, there's crazy mountains and all sorts of crazy shit to check out. If you've ever seen gone with the wind its kind of like that but on acid

**Note:** Don't come to this park if you can't do the trillest of hikes, and don't come in April.

## Adventures

### Hike Through Dopeness Pass

![trail photo](maybe-have-an-image-if-possible-to-sell-them.jpeg)

Yo this hike is sick! You gotta do it, but it's like 9 miles long so be careful.

### Kayak Twisted Canyon

![trail photo](maybe-have-an-image-if-possible-to-sell-them.jpeg)

## Hot Tips

### Rental Car

The Hertz at the Havasupai airport has a manager named Ron and he will hook you up if you say you know Donny Falcone.

### Food

Don't eat the quesadilla at the tortilla factory down the street if you dont want to die
```

I think the Why (+ Why not, for some instances)/What/How is a pretty good way of relaying all available info in an easy way.

Thoughts?

**Comment by yocontra** [comment]
I added two sample files into the _meta folder

**Comment by retlehs** [comment]
:+1: updated with that format, some images, and jeep rental info. any feedback on the images/organization of the files?

File: north_america/united_states/colorado/parks/great_sand_dunes.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+![Sandboarding at the Great Sand Dunes, photo by NPS/Patrick Myers](great-sand-dunes-sandboarding.jpg)
+
+Sandboards and sand sleds are available year-round for rent or purchase at [Kristi Mountain Sports](http://www.slvoutdoor.com/) in Alamosa.
+
+From April thru October, sandboards and sand sleds are available for rent at the [Oasis Store](http://www.greatdunes.com/) just outside the park entrance.
+
+### Backcountry Camping
+
+Free camping, with a permit, is allowed anywhere in the 30-square-mile dunefield outside of the day use area.
```
Comment (yocontra) [inline north_america/united_states/colorado/parks/great_sand_dunes.md:38]: link to get a permit?

**Comment by yocontra** [comment]
Hmm not sure where images should go. Maybe make a _img folder in the root and follow a naming scheme like `united_states_colorado_parks_great_sand_dunes-1.png` ?

File: north_america/united_states/colorado/parks/great_sand_dunes.md (hunk 2 above)
Comment (retlehs) [inline north_america/united_states/colorado/parks/great_sand_dunes.md:38]: no link but added info

**Comment by retlehs** [comment]
done + squashed

**Pull request merged**